core/domain/platform_parameter_domain.py

```diff
@@ -103,21 +99,10 @@ def app_version(self):
 
         Returns:
             str|None. The version of native application, e.g. '1.0.0',
-            None if the client type is web.
+            None if the platform type is Web.
```

Comment:
Please add a TODO on #11208 here, too, since this documentation will need to be updated.

Reply:
Done.